packages/tooltip/style/index.css

```diff
@@ -8,21 +8,17 @@
   border: var(--jp-border-width) solid var(--jp-border-color1);
   font-size: var(--jp-ui-font-size0);
   box-shadow: 0px 1px 6px rgba(0, 0, 0, 0.2);
-  max-width: 500px;
-  overflow: auto;
+  max-width: 650px;
```

Comment:
For wide notebooks, this is great, but if you have two side by side notebooks, the tooltip goes off the dockpanel on the R most notebook. Can we set this size in a way that scales it down when needed? I guess the trigger isn't multiple notebooks, but anytime the tooltip goes off the R side of the screen.